src/renderer/Drawable.ts

```diff
@@ -379,4 +383,35 @@ export default class Drawable {
 
     return this._matrix;
   }
+
+  public getSpriteScale(): number {
+    if (this._sprite instanceof Stage) {
+      return 1;
+    } else {
+      const { size } = this._sprite;
+      if (typeof size !== "number") {
+        this._warnBadSize(typeof size, "0%");
```

Comment:
I believe Scratch tries to cast `size` to a number. Maybe out of scope for this PR, but add a TODO for casting it to a number using Scratch rules? (Out of scope because we should probably do this for other numeric properties as well)

Reply:
Part of #202, yes.

Reply:
Point re: other numeric properties is not own issue yet, but addressed in bullet point:

> It might be reasonable to address this separately from general size boundaries, because e.g. it's probably possible to set other properties to invalid values (with user code) too, and those at least in certain cases would cause fairly brutal errors. We could print warnings in the setter, etc, but the problem isn't unique to size (or related to size boundaries, apart from being treated as 0%).